andromeda/tests/test_satellitedata.py

```diff
@@ -12,10 +12,10 @@
 def sample_rgb_dataframe():
     columns = [LAT_NW, LON_NW, LAT_SE, LON_SE, "Number"]
     data = [
-        (40.332624,	-74.749758,	40.313732,	-74.707701, 111),
+        (40.332624,	-74.749758,	40.313732,	-74.707701, 111,),
```

Comment:
```suggestion
        (40.332624,	-74.749758,	40.313732,	-74.707701, 111),
```

Reply:
Thanks for finding this. I'll incorporate this into the next PR.